components/rmrk/Create/SimpleMint.vue

```diff
@@ -111,7 +111,7 @@
             data-cy="input-batch-address" />
         </NeoField>
         <BasicSlider
-          v-model="distribution"
+          :model-value="distribution"
```

Comment:
This change would break something and then the variable `distribution` would not be changed any more.

Reply:
What solution do you suggest? when using v-model it tells me that I'm missing required prop

Reply:
We could try to add props `@change="sliderChangeHandler"` on it, and we could listen to the update event from `BasicSlider ` component.

Reply:
removed <BasicSlider />, now it's cleaner and it works